src/pages/Main.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react'
+import Display from "../components/Display"
+import Filters from '../components/Filters'
+let rawData = require("../data/data.json")
+
+const Main = () => {
+    const [defaultTag, setDefaultTag] = useState("Please select a category");
```

Comment:
default does not make sense here, better to rename it to tag and setTag, and set it to an empty string or some value from tag